token/js/examples/transferHook.ts

```diff
@@ -0,0 +1,53 @@
+import {
+    clusterApiUrl,
+    sendAndConfirmTransaction,
+    Connection,
+    Keypair,
+    PublicKey,
+    SystemProgram,
+    Transaction,
+    LAMPORTS_PER_SOL,
+} from '@solana/web3.js';
+
+import {
+    ExtensionType,
+    createInitializeMintInstruction,
+    createInitializeTransferHookInstruction,
+    getMintLen,
+    TOKEN_2022_PROGRAM_ID,
+    updateTransferHook,
+} from '../src';
+
+(async () => {
+    const payer = Keypair.generate();
+
+    const mintAuthority = Keypair.generate();
+    const mintKeypair = Keypair.generate();
+    const mint = mintKeypair.publicKey;
+
+    const extensions = [ExtensionType.TransferHook];
+    const mintLen = getMintLen(extensions);
+    const decimals = 9;
+    const programId = new PublicKey('7N4HggYEJAtCLJdnHGCtFqfxcB5rhQCsQTze3ftYstVj');
```

Comment:
nit: same here, can we have `transferHookProgramId` and `newTransferHookProgramId` for the update?